python/cugraph/cugraph/tests/mg/test_mg_connectivity.py

```diff
@@ -32,7 +32,8 @@ def setup_function():
     gc.collect()
 
 
-IS_DIRECTED = [True, False]
+# Directed graph is not currently supported
+IS_DIRECTED = [False]
```

Comment:
You could keep this containing both True and False, and update the test to ensure it properly raises the right exception when a directed graph is used.